src/develop/components/CalloutDoc.js

```diff
@@ -0,0 +1,79 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+var React = require('react');
+var jsxToString = require('jsx-to-string');
+var DocsArticle = require('../../DocsArticle');
+var Callout = require('../../modules/Callout');
+
+Callout.displayName = 'Callout';
+
+var inline = "<Callout ... />";
+
+var CalloutDoc = React.createClass({
+
+  contextTypes: {
+    routePrefix: React.PropTypes.string.isRequired
+  },
+
+  _renderCode(name, jsx) {
+    return (
+      <div>
+        <h3>{name}</h3>
+        <div className="example">
+          {jsx}
+        </div>
+        <pre><code className="html hljs xml">
+          {jsxToString(jsx)}
+        </code></pre>
+      </div>
+    );
+  },
+
+  render: function() {
+    var simpleCallout = (
+      <Callout
+        thumbnail="/docs/img/Case_Study_image.png"
+        label="Case Study"
+        heading="The Key Steps to Reducing Software Spend"
+        description="HPE Software Licensing and Management Solutions can help you optimize your software investments through control of complex negotiations and renewal processes"
+        link="#"
+      />
+    );
+
+    return (
+      <DocsArticle title="Callout" colorIndex="neutral-3">
+
+        <p>The Callout module.</p>
+        <pre><code className="html hljs xml">{inline}</code></pre>
+
+        <section>
+          <h2>Callout Options</h2>
+          <dl>
+            <dt><code>thumbnail            {'{url}'}</code></dt>
+            <dd>Url path to image. Required.</dd>
+            <dt><code>label                {'{string}'}</code></dt>
```

Comment:
Is this what the designers call this line of text? Just wondering, because it may cause confusion as the prop `label` is used in other components for different things with different meanings in Grommet core. If you haven't, can you please check with the designers to see what they would like us to call this line of text or if there's a better name for it?